pkg/util/version/version.go

```diff
@@ -12,13 +12,12 @@ var (
 	SMCP_2_2 = ParseVersion("v2.2")
 	SMCP_2_3 = ParseVersion("v2.3")
 	SMCP_2_4 = ParseVersion("v2.4")
-
-	OCP_4_9  = ParseOCPVersion("4.9.0")
-	OCP_4_10 = ParseOCPVersion("4.10.0")
-	OCP_4_11 = ParseOCPVersion("4.11.0")
-	OCP_4_12 = ParseOCPVersion("4.12.0")
-	OCP_4_13 = ParseOCPVersion("4.13.0")
-	OCP_4_14 = ParseOCPVersion("4.14.0")
+	OCP_4_9  = ParseVersion("4.9.0")
+	OCP_4_10 = ParseVersion("4.10.0")
+	OCP_4_11 = ParseVersion("4.11.0")
+	OCP_4_12 = ParseVersion("4.12.0")
+	OCP_4_13 = ParseVersion("4.13.0")
+	OCP_4_14 = ParseVersion("4.14.0")
 
 	VERSIONS = []*Version{
```

Comment:
Remove this field and remove the function `IsSupported()` (it's not used anywhere) and move `GetPreviousVersion` to smcp_install_test.go, since it's only used there.

It doesn't make sense to have `GetPreviousVersion()` defined in version.go, since `Version` no longer represents only the SMCP version, but is now a generic semantic version object. For the same reason, you should also move the `SMCP_x_y` and `OCP_x_y` constants somewhere else. Maybe moving them simply to `smcp_version.go` and `ocp_version.go` is enough.

Reply:
Done